cmake/checks/check_01_cpu_features.cmake

```diff
@@ -297,7 +297,7 @@ ENDIF()
 #
 
 IF(DEAL_II_WITH_CUDA)
-  SET(DEAL_II_COMPILER_VECTORIZATION_LEVEL 0)
+  SET(DEAL_II_VECTORIZATION_WIDTH_IN_BITS 0)
```

Comment:
unrelated to this PR but does this mean that CPU vectorization is disabled when compiling with CUDA support?

Reply:
Yes, we had problems with the CUDA compiler when we had intrinsics on that were difficult to resolve, see #7655 and #7542.